src/components/Book.js

```diff
@@ -1,18 +1,43 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { removeBook } from '../actions/index';
 
-const Book = ({ id, title, category }) => (
-  <tr>
-    <td>{id}</td>
-    <td>{title}</td>
-    <td>{category}</td>
-  </tr>
-);
+class Book extends React.Component {
+  constructor(props) {
+    super(props);
+    this.bookRemover = this.bookRemover.bind(this);
+  }
+
+  bookRemover() {
+    const { id, removeOldBook } = this.props;
+    removeOldBook(id);
+  }
+
+  render() {
+    const { id, title, category } = this.props;
+    return (
+      <tr>
+        <td>{id}</td>
+        <td>{title}</td>
+        <td>{category}</td>
+        <td><input type="submit" value="X" onClick={this.bookRemover} /></td>
+      </tr>
+    );
+  }
+}
 
 Book.propTypes = {
   id: PropTypes.number.isRequired,
   title: PropTypes.string.isRequired,
   category: PropTypes.string.isRequired,
+  removeOldBook: PropTypes.func.isRequired,
 };
 
-export default Book;
+const mapDispatchToProps = dispatch => ({
+  removeOldBook: state => {
+    dispatch(removeBook(state));
+  },
+});
+
+export default connect(null, mapDispatchToProps)(Book);
```

Comment:
- [ ] Since the Book component is a presentational component, therefore, does not connect with Redux. The handleBookRemove should be implemented in the BookList component and passed to the Book component as a callback prop.